### DISCUSSION TITLE
Tabbing in/out issue.

### DISCUSSION THREAD

**Discussion by TeddybearEnthusiast** [ppy/osu#16015] [Q&A]
Unimportant but repeatable bug where alt-tabbing after completing a beatmap in the interval before the game takes you to the score screen causes the game to lock in this state. Upon tabbing back in, the game will only take you to the score screen upon registering a button press.

All things considered probably not really worth fixing because seems totally innocuous, but thought I'd put it here just in case.

**Comment by peppy** [comment]
Please provide logs as per https://github.com/ppy/osu/discussions/15188. A video showing what you mean would also be helpful.
